scripts/eventTest.ts

```diff
@@ -0,0 +1,39 @@
+// TO RUN: npx ts-node scripts/eventTest.ts
```

Comment:
Can you guys delete this file from the PR? I think ideally we wanna only keep files that the webapp will need (so omit testing files)